libyul/optimiser/Suite.cpp

```diff
@@ -310,6 +315,10 @@ void OptimiserSuite::validateSequence(string_view _stepAbbreviations)
 			nestingLevel--;
 			assertThrow(nestingLevel >= 0, OptimizerException, "Unbalanced brackets");
 			break;
+		case ':':
+			assertThrow(nestingLevel == 0, OptimizerException, "Cleanup delimiter may only be placed at nesting level zero");
```

Comment:
Also, the current message is not bad, but I think that for the user it might be clearer to emphasize the brackets rather than the exact nesting level:
```suggestion
			assertThrow(nestingLevel == 0, OptimizerException, "Cleanup delimiter cannot be nested inside the brackets");
```

Reply:
Cleanup sequence delimiter it is.